### PR TITLE
💄 {refactor} Created Bot Choose file input  #120

### PR DESCRIPTION
**Pull request by faisoabdirisak**
# Description

Edited Choose bot file input to the required design.
Redesigned the create modal.

Fixes # (issue) -#120

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# Screenshot (optional)

Screenshot (750)

Screenshot (786)

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

### CODE REVIEW

File: libs/features/convs-mgr/home/src/lib/services/new-story.service.ts [hunk 1]
```diff
@@ -18,6 +18,7 @@ import { StoriesStore } from "@app/state/convs-mgr/stories";
 export class NewStoryService implements OnDestroy
 {
   private _sbS = new SubSink();
+  image:false;
```
Comment (IanOdhiambo9) [inline libs/features/convs-mgr/home/src/lib/services/new-story.service.ts:21]: ??

**Review by IanOdhiambo9 — CHANGES_REQUESTED** [review]
Find time to schedule a brief call so I can guide you on a strategy to do this.

File: libs/features/convs-mgr/home/src/lib/services/new-story.service.ts (hunk 1 above)
Comment (faisoabdirisak) [reply libs/features/convs-mgr/home/src/lib/services/new-story.service.ts]: done all the changes

File: .vscode/settings.json [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{
+    "cSpell.words": [
+        "convl",
+        "environnment"
+    ]
+}
```
Comment (IanOdhiambo9) [inline .vscode/settings.json:6]: Don't push this

File: apps/conv-learning-manager/.gitignore [hunk 3]
```diff
@@ -1 +1 @@
-src/environments/*
\ No newline at end of file
+src/environments/*
```
Comment (IanOdhiambo9) [inline apps/conv-learning-manager/.gitignore]: Don't push this as well

File: apps/conv-learning-manager/src/main.ts [hunk 4]
```diff
@@ -2,9 +2,9 @@ import { enableProdMode } from '@angular/core';
 import { platformBrowserDynamic } from '@angular/platform-browser-dynamic';
 
 import { AppModule } from './app/app.module';
-
 import { environment } from './environments/environment';
 
+
 if (environment.production) {
   enableProdMode();
 }
```
Comment (IanOdhiambo9) [inline apps/conv-learning-manager/src/main.ts:10]: Don't push this as well

**Review by IanOdhiambo9 — CHANGES_REQUESTED** [review]
Your PR is okay, the updates look good however there are files you've committed that should not be pushed. package.json, package-lock.json, vscode/settings.json, main.ts and your git ignore.

**Pull request closed without merging**